petri/src/vm/mod.rs

```diff
@@ -137,6 +138,12 @@ pub struct PetriVmConfig {
     pub tpm: Option<TpmConfig>,
 }
 
+/// VM configuration that can be changed after the VM is created
```

Comment:
So I understand the intent ... will this eventually include things like disks? (a disk can be added/removed at runtime, but not a controller)